ftp/src/test/java/akka/stream/alpakka/ftp/CommonFtpStageTest.java

```diff
@@ -38,6 +38,10 @@
   Sink<FtpFile, CompletionStage<IOResult>> getMoveSink(Function<FtpFile, String> destinationPath)
       throws Exception;
 
+  default IOResult await(CompletionStage<IOResult> result) throws InterruptedException, java.util.concurrent.ExecutionException, java.util.concurrent.TimeoutException {
+    return result.toCompletableFuture().get(5, TimeUnit.SECONDS);
```

Comment:
Increased timeout for these Java tests.

Reply:
A build broke even with 5 seconds.  Maybe upgrade to 10 before merging?

https://travis-ci.com/akka/alpakka/jobs/277549627#L943